bin/problem.py

```diff
@@ -131,6 +132,17 @@ def _read_settings(self):
             'time_multiplier': 2,
             'time_safety_margin': 1.5,
```

Comment:
What is your plan on implementing / backwards-compatibility regarding [the new spec for `time_multipliers`](https://github.com/Kattis/problem-package-format/blob/master/spec/2023-07-draft.md#problem-timing)? (doesn't need to be implemented right away if you don't feel like it, the spec is still in draft anyway, but just curious :smile: )

Reply:
These keys already existed before this PR :D
but I would propose to map the new keys to the old ones... having them not in a nested dict is much more convenient ^^'